### DISCUSSION TITLE
delegation bug

### DISCUSSION THREAD

**Discussion by Validatrium** [subquery/subql#906] [Frontier TestNet - Bug Reporting]
upon entering amount I want to delegate to my node, button gets gray and amount of delegation reverts to "0", metamask doesn't pop up (screenshots attached)
![screenshot 1](https://user-images.githubusercontent.com/88688112/164701042-0c70687b-f5a8-41a9-9205-497f3973711f.png)
![screenshot 2](https://user-images.githubusercontent.com/88688112/164701036-da17d682-6bfc-499e-9984-b62f6e1c8c65.png)

**Comment by ianhe8x** [comment]
@CaiYiLiang

**Comment by heyacherry** [comment]
@Validatrium Thanks for the issue raise! 
I failed to reproduce the bug and I think the stake/unstake should be working in prod now. Have you got any issues on stake/unstake after our prod has on live?

**Comment by lilian79511** [comment]
This is reported as fixed by @CaiYiLiang. Discussion will be locked.
